internal/cli/cli.go

```
@@ -0,0 +1,222 @@
+		return runPlan(args[1:], stdout, stderr)
+	case "task", "tasks":
+		fmt.Fprintln(stderr, "AGX-UNSUPPORTED-TASK: AGX does not create, assign, or schedule daily Tasks")
+		return exitcode.Unsupported
+	}
+
+	if knownLifecycleCommand(commandName) {
+		fmt.Fprintf(stderr, "AGX-UNSUPPORTED-COMMAND: %q is not implemented in this preview\n", commandName)
+		return exitcode.Unsupported
+	}
+
+	fmt.Fprintf(stderr, "AGX-INVALID-INVOCATION: unknown command %q\n", commandName)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)